src/player.rs

```diff
@@ -68,10 +72,7 @@ impl Plugin for PlayerPlugin {
 }
 
 #[derive(Component)]
-enum Direction {
-	Right,
-	Left,
-}
+struct Player {}
```

Comment:
I originally have this as an enum

```rs
#[derive(Component)]
struct Character {
    Player,
    Enemy
}
```

But I wasn't able to query for `Character::Player` in the bevy system... I guess bevy doesn't like that as a type? Let's figure out if there's a right way to do this, maybe I'm reaching the limits of my rust know-how